sdk/template/Azure.Template/perf/Azure.Template.Perf.csproj

```diff
@@ -0,0 +1,16 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\src\Azure.Template.csproj" />
+    <ProjectReference Include="..\..\..\..\common\Perf\Azure.Test.Perf\Azure.Test.Perf.csproj" />
```

Comment:
We might want to consider defining a property for the path to the project as these long relative paths can sometimes become fragile.

Reply:
Is there an example?

Reply:
Will fix in later PR: https://github.com/Azure/azure-sdk-for-net/issues/16709